.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,11 @@
-<!--- Briefly describe what you're changing. -->
+<!--- Briefly describe what you're changing in the pull request title. -->
```

Comment:
Nitpick: replace with

> Briefly describe your changes in the field above.

It's clear from context that it's a pull-request, and "title" is  less obvious than pointing to a form-field.